tests/next_tests/integration_tests/feature_tests/ffront_tests/test_execution.py

```diff
@@ -325,6 +325,76 @@ def testee(a: cases.IFloatField) -> gtx.Field[[IDim], int64]:
     )
 
 
+@pytest.mark.uses_tuple_returns
+def test_astype_on_tuples(cartesian_case):  # noqa: F811 # fixtures
+    @gtx.field_operator
+    def field_op_returning_a_tuple(
+        a: cases.IFloatField, b: cases.IFloatField
+    ) -> tuple[gtx.Field[[IDim], float], gtx.Field[[IDim], float]]:
+        tup = (a, b)
+        return tup
+
+    @gtx.field_operator
+    def cast_tuple(
+        a: cases.IFloatField,
+        b: cases.IFloatField,
+        a_casted_to_int_outside_of_gt4py: cases.IField,
```

Comment:
```suggestion
        a_asint: cases.IField,
```
The naming was not a serious proposal of mine, but just to signify what it meant. Here it should be clear from the context.